src/test/java/se/sundsvall/byggrintegrator/api/OpeneHtmlResourceTest.java

```diff
@@ -29,6 +29,8 @@ class OpeneHtmlResourceTest {
 	@Autowired
 	private WebTestClient webTestClient;
 
+	private static final String INFO_QUERY_RESPONSE_HEADER = "InfoQueryResponse";
```

Comment:
The constants in the `ResponseEnhancer` could be made public and reused, to avoid the risk of changes either in that class or here leading to test failures.

Reply:
Hm, don't we want the test to actually test that they have the correct value? e.g. if I change the header to "DonaldDuck" the tests will go green but it won't work? :)

Reply:
Yup. Correct thinking